src/assets/stylesheets/info-dialog.scss

```diff
@@ -79,6 +79,10 @@
   }
 }
 
+.dialog__box__contents__body p:first-child {
```

Comment:
we haven't been too crazy about this, but I think technically this breaks BEM -- probably should be a modifier class. your call though if you'd like to refactor!

Reply:
yes, good eye - it does. will fix.

Reply:
the more I think about this, I think pseudo selectors could/should be the exception to the BEM pattern used throughout. let me know, and I can add a `margin-top-less` or something to each first `<p>`.